src/app/pages/attributes-page/attributes.page.ts

```diff
@@ -24,28 +24,20 @@ export class AttributesPage implements OnInit {
 
   }
 
-  getAttributes() {
-    // await this._attributesModelService.getAttributesByUser();
-
-    return [
-        {
-            "phrase": {
-                "adverb": "competitively",
-                "verb": "writes",
-                "preposition": null,
-                "noun": "code"
-            }
-        },
-        {
-            "phrase": {
-                "adverb": null,
-                "verb": "plays",
-                "preposition": null,
-                "noun": "chess"
-            }
-        }
-    ];
-  }
+  //  getAttributes() {
+
+  //   return this._attributesModelService.getAttributesByUser();
+    
+  //  }
+
```

Comment:
The commented out code above is the correct way to return the attributes. There might be more you need to do to make it work, but it is certainly closer to the correct way.